renderdoc/core/core.cpp

```diff
@@ -319,7 +319,7 @@ RenderDoc::~RenderDoc()
   if(m_RemoteThread)
   {
     m_RemoteServerThreadShutdown = true;
-    // don't join, just close the thread, as we can't wait while in the middle of module unloading
```

Comment:
See the [other comment chain](https://github.com/michaelrgb/renderdoc/commit/62a62b4ebed4c43d1d119b0d367c161879d99f08#commitcomment-18690241).